echopype/convert/set_groups_azfp.py

```diff
@@ -93,7 +81,7 @@ def _create_unique_channel_name(self):
         """
 
         serial_number = self.parser_obj.unpacked_data["serial_number"]
-        frequency_number = self.parser_obj.parameters["frequency_number"]
+        frequency_number = self.parser_obj.parameters["frequency_number_phase1"]
```

Comment:
Was this an oversight that you caught now, or a change required by your other recent changes?

Reply:
I caught this now after incorporating `phase number` to a child attribute names as well.

Reply:
Ah, that makes sense. Thanks.